index.bs

```diff
@@ -435,3 +735,6 @@ A file containing a 'pict' track compliant with this profile is expected to list
 <p>The media type <code>"image/avif"</code> is officially registered with IANA and available at: <a href="https://www.iana.org/assignments/media-types/image/avif">https://www.iana.org/assignments/media-types/image/avif</a>.</p>
 
 <h2 id="change-list">Changes since v1.1.0 release</h2>
+
+  - <a href="https://github.com/AOMediaCodec/av1-avif/pull/224">Move section 4 "Auxiliary Image Items and Sequences" to section 4.1 under new section 4 "Other Image Items and Sequences".</a>
+  - <a href="https://github.com/AOMediaCodec/av1-avif/pull/224">Add section 4.2.1 "Sample Transform Derived Image Item" under new section 4.2 "Derived Image Items and Sequences".</a>
```

Comment:
Nit/optional: It may be clearer to say:

> Add section 4.2 "Derived Image Items and Sequences" and section 4.2.1 "Sample Transform Derived Image Item".